### PR TITLE
Components - NavBar

### LINKED ISSUE #10: Create Navbar components
## Description
Provide a clear and concise description of the feature you're suggesting
```
Create 2 navbar components in /src/components

One navbar component is HomeNavbar which follows this design - https://zpl.io/aNNB7zZ
The other navbar component is Navbar which follows this design - https://zpl.io/2jNkoM6
```

### PR DESCRIPTION
**Pull request by GreatGuy96**
```
This pull adds the navbar component in the signed out state for both the home page as well as a separate bar for other pages.
```

Resolves #10

### CODE REVIEW

**Review by TrustedMercury — CHANGES_REQUESTED** [review]

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,15 +1,42 @@
...
 
   return (
-    <nav className="flex flex-row items-center justify-between">
-      <div className="flex flex-row items-center justify-start">
-        <span>haven</span>
-      </div>
-      <div className="flex flex-row items-center justify-end">
-        <span>login signup</span>
+    <nav>
+      {home ? (<></>) : (
```
[the remark this answers is not in the document]
Comment (GreatGuy96) [reply src/components/Navbar.js]: Fixed

File: src/components/Navbar.js [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
+          <>
+            <div className="absolute bg-no-repeat bg-blue-500 w-full h-20 z-0 top-0"/>
+          </>
+        )}
+      <div className=" flex flex-row items-center justify-between top-0">
+        <div className="flex flex-row items-center justify-start z-10 ml-10">
+          <Link href = "/"><a>
```
[the remark this answers is not in the document]
Comment (GreatGuy96) [reply src/components/Navbar.js]: Fixed

File: src/components/Navbar.js [hunk 3]
```diff
@@ -1,15 +1,42 @@
...
+            <Image
+              alt="Haven Logo"
+              src="/images/logo-wordmark-white.svg"
+              width={219}
+              height={70}
```
[the remark this answers is not in the document]
Comment (GreatGuy96) [reply src/components/Navbar.js]: I tried with 256 and 128. You said 128 was too small and 256 was too big. You kept being picky until I got to 219. I think it's good as is.

File: src/components/Navbar.js [hunk 4]
```diff
@@ -1,15 +1,42 @@
...
+              layout="intrinsic"
+            />
+          </a></Link>
+        </div>
+        <div className="flex flex-row items-center justify-end z-10 mr-10">
+          <button className="flex justify-center px-5 py-1.5 bg-gray-300 z-10 rounded mr-4 hover:bg-gray-400 active:bg-gray-200 focus:outline-none">
```
[the remark this answers is not in the document]
Comment (GreatGuy96) [reply src/components/Navbar.js]: - Fixed - Yeah I got confused with the focus and active things, I don't know much about those
- Fixed - K
- Fixed - Yeah that was for testing, I forgot to remove them

File: src/components/Navbar.js [hunk 5]
```diff
@@ -1,15 +1,42 @@
...
+            <span className="font-inter font-semibold text-center select-none text-white text-lg">Log In</span>
+            
+          </button>
+          <button className="flex justify-center px-5 py-1.5 bg-blue-350 z-10 rounded mr-4 hover:bg-blue-450 active:bg-blue-300 focus:outline-none">
```
[the remark this answers is not in the document]
Comment (GreatGuy96) [reply src/components/Navbar.js]: Fixed

**Review by TrustedMercury — APPROVED** [review]
Looks good!

**Pull request merged**